catalog/spatial/registry/registry-rest-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -38,7 +38,7 @@
         <property name="registryReportMapBuilder" ref="registryReportMapBuilder"/>
     </bean>
 
-    <jaxrs:server id="registryRestEndpointService" address="/registries">
+    <jaxrs:server id="registryRestEndpointService" address="/internal/registries">
         <jaxrs:serviceBeans>
```

Comment:
you should modify the path used in `testRestEndpoint` for the itest `TestRegistry` (line 278)